src/docs/build-posix.dox

```diff
@@ -21,7 +21,7 @@ We also suggest <a href="https://ninja-build.org/">Ninja</a> and
 First, clone the repository:
```

Comment:
General note: The commit messages say WT-13506 not WT-13560. This should be ok as we use the PR title for merge.

Reply:
Good catch. It was a typo in the commit title.